spec/unit/packagers/msi_spec.rb

```diff
@@ -316,6 +316,23 @@ module Omnibus
       end
     end
 
+    describe '#fast_msi' do
```

Comment:
ok. yeah. looking at it more closely i see they don't call the `build` method which is where the bulk of the fastmsi stuff is.